builtin/providers/aws/resource_aws_vpc_migrate_test.go

```diff
@@ -0,0 +1,43 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAWSVpcMigrateState(t *testing.T) {
+	cases := map[string]struct {
+		StateVersion int
+		ID           string
+		Attributes   map[string]string
+		Expected     string
+		Meta         interface{}
+	}{
+		"v0_1": {
+			StateVersion: 0,
+			ID:           "some_id",
+			Attributes: map[string]string{
+				"assign_generated_ipv6_cidr_block": "true",
```

Comment:
it might make sense to add a test without this assignment (for users upgrading from prior to 0.9)